transform/mattermost-analytics/models/marts/web_app/platform/eg_performance_events.sql

```diff
@@ -0,0 +1,35 @@
+{{
+    config({
+        "materialized": "incremental",
+        "cluster_by": ['event_date']
+    })
+}}
+
+WITH performance_rc AS (
+    SELECT 
+        'int_mm_telemetry_rc_performance_events' AS _source_relation,
+        {{dbt_utils.star(ref('int_mm_telemetry_rc_performance_events'))}}
+     FROM
+       {{ ref('int_mm_telemetry_rc_performance_events') }}
+{% if is_incremental() %}
+    WHERE received_at_date >= (
```

Comment:
Incremental strategy should rely on `received_at`.

Reply:
Updated to `received_at`

Reply:
Both ends of the expression in the `WHERE` clause must use the same column (`received_at`).

Reply:
Updated both clauses.